### PR TITLE
Add PostCSS

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Hi there,

First of all, thanks for this awesome list!
I just saw that PostCSS is missing, so I added it.

Have a nice day!

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please read and adhere to the [contributing guidelines](../blob/master/contributing.md).

File: readme.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Charting](https://github.com/zingchart/awesome-charting)
 - [Ionic Framework 2](https://github.com/candelibas/awesome-ionic2)
 - [Chrome DevTools](https://github.com/ChromeDevTools/awesome-chrome-devtools)
-
```
Comment (sindresorhus) [inline readme.md]: Don't remove the newline

**Comment by sindresorhus** [comment]
- `Table of Contents:` => `Table of Contents`
- All the grunt and gulp plugin names should be fully lowercased
- `How to contribute:` => `Contribute`

**Comment by sindresorhus** [comment]
// @ai

**Comment by DanielMSchmidt** [comment]
Sorry for the trouble, just opened a PR on the other project to make it compliant. Thank you so much for your awesome open source work!

**Comment by ai** [comment]
LGFM

**Comment by sindresorhus** [comment]
Some more:

- It should not use the MIT. That's a code license. See: https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license
- Needs the awesome badge: https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge
- I would like to see links to community resources (Example: https://github.com/sindresorhus/awesome-electron#community).

-

// @jjaderg

**Comment by DanielMSchmidt** [comment]
Added the things mentioned in a PR, I will give you feedback as it gets merged. Thank you so much for your time!

**Comment by DanielMSchmidt** [comment]
The [PR was merged](https://github.com/jjaderg/awesome-postcss/pull/12), hope the document suits your needs now 👍

**Comment by sindresorhus** [comment]
https://github.com/jjaderg/awesome-postcss/blob/master/license.md needs to be updated or removed regarding the license change. And there needs to be a space between the build badge and the awesome badge. Otherwise looks good.

**Pull request merged**